apps/daffio/src/app/content/homepage/component/homepage.component.scss

```diff
@@ -45,21 +47,32 @@
     grid-gap: 15px;
 
     > * {
-      width: 100%;
+      @include breakpoint(big-tablet) {
+        width: 150px;
+      }
+
+      @include breakpoint(small-laptop) {
+        width: 210px;
+      }
     }
   }
 
   &__list {
+    z-index: 2;
+    text-align: center;
```

Comment:
I'm not a fan of the text-align center for lists. I think it's quite a bit more difficult to read.

Reply:
Agree. Removed in 44dbc2e